client/src/components/views/ProfilePage/LoginPage.tsx

```diff
@@ -1,7 +1,81 @@
 import React from "react";
+import { BASE_URL } from "../../../hooks/UseFetch";
+import { useNavigate } from "react-router";
+import { signInSchema } from "../../../shared/inputHandlers/LoginInputHandler";
+import { useFormik } from "formik";
+import axios from "axios";
 
 const LoginPage = () => {
-  return <div>LoginPage</div>;
+  const signInFormHandler = useFormik({
+    initialValues: {
+      email: "",
+      password: "",
+    },
+    validationSchema: signInSchema,
+    onSubmit: (values) => alert(JSON.stringify(values)),
+  });
+
+  const navigate = useNavigate();
+  const navigation = () => {
+    navigate("/user/your-offers");
+  };
+
+  const onSubmitHandler = async (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+    signInFormHandler.handleSubmit;
+
+    await axios({
+      method: "post",
+      url: BASE_URL + "/users/login",
+      data: {
+        email: signInFormHandler.values.email,
+        password: signInFormHandler.values.password,
+      },
+    }).then((response) => {
+      const correctResponse = response.status == 200;
+      if (!correctResponse) {
+        throw Error("Incorrect response, check credentials");
```

Comment:
maybe if we have wrong credentials, we will return a component from react toastify?